microsoft-edge/accessibility/test.md

```diff
@@ -18,7 +18,41 @@ Use the following tools and testing procedures to evaluate your website for acce
 <!-- ====================================================================== -->
 ## Accessibility testing in DevTools
 
```

Comment:
unable to navigate.  might need index entry in toc
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/b74704e5-5aa7-4ac9-b3d8-aaa8dd1c7b2a)

Reply:
made Accessibility TOC entries match top-of-article titles